docs/monitors/df.md

```diff
@@ -0,0 +1,79 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# df
+
+
+This monitor reports metrics about free disk space.
+
+```yaml
+monitors:
+ - type: df
+```
+
+
+Monitor Type: `df`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/df)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: **No**
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `procFSPath` | no | `string` | The path to the proc filesystem. Useful to override in containerized environments.  (Does not apply to windows) (**default:** `/proc`) |
+| `ignoreSelected` | no | `bool` | If true, the filesystems selected by `fsTypes` and `mountPoints` will be excluded and all others included. |
```

Comment:
I hate this aspect of the collectd df plugin, and it is really confusing.  Let's figure out a better way to configure this.

Reply:
ack